Scripts/extras/pull_and_build_from_git.py

```diff
@@ -780,10 +781,20 @@ def build_and_install_custom(self):
         
         env_to_use = self.create_custom_env()
         custom_build_cmds = self.platform_config.get('custom_build_cmd', [])
-        for custom_build_cmd in custom_build_cmds:
+        if custom_build_cmds:
+
+            if len(custom_build_cmds)>1:
+                custom_build_cmd_args = ' '.join([f'"{cmd.strip()}"' if " " in cmd else f'{cmd.strip()}' for cmd in custom_build_cmds[1:]])
+                full_custom_build_cmd = f'{custom_build_cmds[0].format(python=sys.executable)} {custom_build_cmd_args}'
+            else:
+                full_custom_build_cmd = custom_build_cmds[0].format(python=sys.executable)
+
+
+            print(f"CUSTOM BUILD COMMAND: {full_custom_build_cmd}")
+
             # Support the user specifying {python} in the custom_build_cmd to invoke
             # the Python executable that launched this build script
-            call_result = subprocess.run(custom_build_cmd.format(python=sys.executable),
+            call_result = subprocess.run(full_custom_build_cmd,
```

Comment:
In python2 it was recommended to pass in a single string when using `shell=True` ( https://docs.python.org/2/library/subprocess.html)
```
The shell argument (which defaults to False) specifies whether to use the shell as the program to execute. If shell is True, it is recommended to pass args as a string rather than as a sequence
```
I don't know if this is still the case for python3. I will check.

But I do see an alternative to this: https://docs.python.org/3/library/shlex.html